config/kubernetes/org.yaml

```diff
@@ -1734,9 +1735,12 @@ teams:
     - enj
     - joelsmith
     - micahhausler
+    - natherz97
+    - puerco
     - ritazh
     - SaranBalaji90
     - tabbysable
+    - Vyom-Yadav
```

Comment:
@SaranBalaji90, again for tracking purpose – could you please update the commit message to say something like `add new SRC members – puerco, Vyom-Yadav`? Thanks!